crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/StatelessAssessmentService.java

```diff
@@ -43,88 +39,86 @@ public StatelessResult execute(Assessment assessment,
                                    Map<AgeRange, Integer> childGroupings,
                                    List<Income> income,
                                    List<Outgoing> outgoings) {
+        final var assessmentCriteriaEntity = assessmentCriteriaService.getAssessmentCriteria(
+                assessment.getAssessmentDate(), assessment.getHasPartner(), false
+        );
+        var initialAnswer = initialResult(
+                childGroupings,
+                assessmentCriteriaEntity,
+                assessment.getCaseType(),
+                assessment.getMagistrateCourtOutcome(),
+                income,
+                null);
```

Comment:
More of a query for the future - Is there a plan to allow the new work reason to be an optional input to the stateless service? This will be required if we want MAS to call SMAS under the hood.

Reply:
Yes I think so - the change was included at one point, but removed from this PR to focus on the refactoring